packages/@webex/plugin-meetings/test/unit/spec/reconnection-manager/index.js

```diff
@@ -10,7 +11,74 @@ const {assert} = chai;
 chai.use(chaiAsPromised);
 sinon.assert.expose(chai.assert, {prefix: ''});
 
+
 describe('plugin-meetings', () => {
+  describe('ReconnectionManager.reconnectMedia', () => {
+    it('uses correct TURN TLS information on reInitiatePeerconnection', async () => {
+      const fakeMediaConnection = {
+        initiateOffer: sinon.stub().resolves({})
+      };
+      const fakeMeeting = {
+        peerConnection: sinon.stub().returns(fakeMediaConnection),
```

Comment:
this one is not needed and misleading. meeting object has peerConnection inside meeting.mediaProperties which you already mock in line 41